toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -1,5 +1,5 @@
 filesystem-1.1-11.cm2.aarch64.rpm
-kernel-headers-5.15.63.1-2.cm2.noarch.rpm
```

Comment:
Also make these updates to the full toolchain manifests (the pkggen core is a subset of the full toolchain list)

Reply:
Added. Thank you